homeassistant/components/ezviz/alarm_control_panel.py

```diff
@@ -0,0 +1,85 @@
+"""Support for Ezviz alarm."""
```

Comment:
Please don't add this platform in this PR. Let's focus on getting this PR merged, ie no new features just complete the minimum needed to have a working flow and tests for that.

Reply:
Removed.